students/AleGuardia/Lesson06/test.py

```diff
@@ -0,0 +1,141 @@
+# tests for Mailroom_four.py by Alejandro Guardia
+
+from mailroom_four import return_average
+from mailroom_four import return_donors
+from mailroom_four import return_label
+from mailroom_four import return_report
+from mailroom_four import add_donor
+from mailroom_four import add_donation
+from mailroom_four import prompt_donors
+from mailroom_four import print_report
```

Comment:
note that you can do this in single import line:
`from mailroom_four import print_report, add_donor`

or 

```
import mailroom_four


mailroom_four.print_donor  # usage
```